lib/util/fs/proxies.ts

```diff
@@ -1,5 +1,5 @@
 import fs from 'fs-extra';
-import type { MoveOptions, WriteFileOptions } from 'fs-extra';
+import type { WriteFileOptions } from 'fs-extra';
 
 // istanbul ignore next
```

Comment:
shouldn't we add tests instead of remove the spec file completely?

Reply:
I'll test remaing files once they've refactored